packages/botfuel-dialog/src/dialogs/qnas-dialog.js

```diff
@@ -34,11 +34,11 @@ class QnasDialog extends Dialog {
   }
 
   /** @inheritDoc */
-  async execute(adapter, userMessage, messageEntities) {
-    logger.debug('execute', userMessage, messageEntities);
-    const extraData = await this.dialogWillDisplay(userMessage, messageEntities);
-    const answers = messageEntities;
-    const dialogData = { messageEntities, answers, extraData };
+  async execute(adapter, userMessage, data) {
+    logger.debug('execute', userMessage, data);
+    const { answers } = data;
```

Comment:
you could write:
logger.debug('execute', userMessage, data);
const extraData = await this.dialogWillDisplay(userMessage, data);
await this.display(adapter, userMessage, {...data, extraData});